scripts/packaging/Makefile

```
@@ -68,7 +68,8 @@ ISCC_FORMATTER = iscc_formatter.exe
 endif
 
 TARGET_URL = $(shell cat $(WEBOTS_HOME_PATH)/scripts/packaging/webots_version.txt)
-ifeq ($(shell git tag | grep $(TARGET_URL)),)
+TAGS = $(shell git fetch --all --tags)
+ifeq ($(shell git tag --points-at HEAD | grep $(TARGET_URL)),)
```

I believe we have to use `:=` instead of `=` otherwise, since `TAGS` is never used, the `git fetch` will never be executed. See https://stackoverflow.com/a/6283363/810268
```suggestion
TAGS := $(shell git fetch --all --tags)
ifeq ($(shell git tag --points-at HEAD | grep $(TARGET_URL)),)
```
Also, in general for `$(shell ...)` commands, it's better to use `:=` (more efficient, less error prone).